internal/ethapi/api.go

```diff
@@ -1602,14 +1602,20 @@ func (args *SendTxArgs) toTransaction() *types.Transaction {
 	return types.NewTx(data)
 }
 
+// isEIP1820Transaction checks if the given transaction is the EIP1820 transaction. This is allowed
+// to be submitted even if it is not using EIP-155
```

Comment:
Please add a link to the [EIP](https://eips.ethereum.org/EIPS/eip-1820). To a casual reader, without reading the eip itself, this looks extremely strange.

Reply:
I've added link and a bit more information.